catalog/ui/search-ui/standard/src/main/doc/_contents/help-contents.adoc

```diff
@@ -279,17 +279,14 @@ the download link.
 
 === Workspaces
 
-Each user can create multiple workspaces and assign each of them a descriptive name. Each workspace
-can contain multiple <<Save a Search, saved searches>> and contain multiple saved records
-(metacards). Workspaces are saved for each user and are loaded when the user logs in. Workspaces
-and their contents are persisted, so they survive if {branding} is restarted. Currently, workspaces
-are private and cannot be viewed by other users.
+Each user can create multiple workspaces and assign each of them a descriptive name.
+Each workspace can contain multiple <<Save a Search, saved searches>> and contain multiple saved records.
+Workspaces are saved for each user and are loaded when the user logs in.
+Workspaces and their contents are persisted, so they survive if {branding} is restarted. Within the Standard Search UI, workspaces are private and cannot be viewed by other users.
```

Comment:
This help page is specific to the older Standard Search UI. This section was revised to clarify a different behavior between the two UIs.